src/schemas/21-526EZ/schema.js

```diff
@@ -36,9 +40,46 @@ const disabilitiesBaseDef = {
   }
 };
 
-
-// Extracted to enable easy adding of properties for forwardingAddress
-const addressDef = definitions.pciuAddress;
+const addressBaseDef = {
+  type: 'object',
+  required: ['country', 'city', 'addressLine1'],
+  properties: {
+    country: {
+      type: 'string',
+      'enum': pciuCountries,
```

Comment:
Are we interested in using 'United States' as the label for USA?

Reply:
Yeah, this has gone back and forth a few times. I confirmed with Tim that EVSS only accepts 'USA', and it seems kind of silly to create a value / label map for just one country. Not ideal, but we do it elsewhere across the site, and I've set the default to 'USA' so at least US-based veterans don't have to go hunting through the list.